src/game_api/rpc.cpp

```diff
@@ -920,7 +917,7 @@ void set_kapala_hud_icon(int8_t icon_index)
 
     if (instruction_offset == 0)
     {
-        auto state = State::get();
+        const auto& state = State::get();

```

Comment:
Did you run some sort of clang-tidy on this?

Reply:
it was throwing warning (or check? they are all called warnings in MSVC i guess) C26820

Reply:
also the state variable is not used here, does the get() function does something else? maybe i should remove it completely?

Reply:
i will fix it more in my warning fix PR, this does nothing anyway, can pass for now